applications/rag/README.md

```diff
@@ -1,192 +1,210 @@
-    * Choose a custom k8s namespace and service account to be used by the application.
-    * Set `create_network = true` if you want to create a new VPC network
+2. Edit `workloads.tfvars` to set your project ID, location, cluster name, and GCS bucket name. Ensure the `gcs_bucket` name is globally unique (add a random suffix). Optionally, make the following changes:
+    * (Optional) Set a custom `kubernetes_namespace` where all k8s resources will be created.
+    * (Recommended) [Enable authenticated access](#configure-authenticated-access-via-iap) for JupyterHub, frontend chat and Ray dashboard services:
+    * (Not recommended) Set `create_cluster = false` if you bring your own cluster. If using a GKE Standard cluster, ensure it has an L4 nodepool with autoscaling and node autoprovisioning enabled.
+    * (Not recommended) Set `create_network = false` if you bring your own VPC. Ensure your VPC has Private Service Connect enabled as described above.
 
 3. Run `terraform init`
 
 4. Run `terraform apply --var-file workloads.tfvars`
 
-5. If you set `jupyter_domain`, `frontend_domain` or `ray_dashboard_domain`, set up your DNS service to point to the public IP
-    * run command `terraform output frontend_ip_address` to get the public ip address of frontend, and add an A record in your DNS configuration to point to the public IP address.
-    * run command `terraform output jupyterhub_ip_address` to get the public ip address of jupyterhub, and add an A record in your DNS configuration to point to the public IP address.
-    * run command `terraform output ray_dashboard_ip_address` to get the public ip address of ray dashboard, and add an A record in your DNS configuration to point to the public IP address.
+# Generate vector embeddings for the dataset
 
-    If the DNS service of your domain is managed by [Google Cloud DNS managed zone](https://cloud.google.com/dns/docs/zones), we have two ways to add the A record:
-    1. Go to https://console.cloud.google.com/net-services/dns/zones, select the zone and click ADD STANDARD, fill in your domain name and public IP address.
-    2. Run command: `gcloud dns record-sets create <domain address>. --zone=<zone name> --type="A" --ttl=<ttl in seconds> --rrdatas="<public ip address>"`
+This section generates the vector embeddings for your input dataset. Currently, the default dataset is [Netflix shows](https://www.kaggle.com/datasets/shivamb/netflix-shows). We will use a Jupyter notebook to run a Ray job that generates the embeddings & populates them into the `pgvector` instance created above.
 
-6. Optionally, enable Cloud Data Loss Prevention (DLP)
+Set env vars (change to the namespace in `workloads.tfvars` if using a custom `kubernetes_namespace`):
```

Comment:
I think the previous wording was less confusing:

`Set your namespace from `workloads.tfvars`:`